service/device/registry/test/src/test/resources/features/DeviceRegistry.feature

```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2017, 2019 Eurotech and/or its affiliates and others
+# Copyright (c) 2020 Eurotech and/or its affiliates and others
```

Comment:
I see this only now - please change header to " * Copyright (c) 2017, 2020 Eurotech and/or its affiliates and others"